contextualized/regression/tests.py

```diff
@@ -61,10 +61,10 @@ def _quicktest(self, model, univariate=False, correlation=False):
         dataloader = None
         trainer = None
         if correlation:
-            dataloader = model.dataloader(self.C, self.X, batch_size=self.batch_size)
+            dataloader = model.dataloader(self.C, self.X, batch_size=self.batch_size, num_workers=2)
```

Comment:
Please add a specific unit test for which tries multiple values of `num_workers`.

That will probably include modifying `_quicktest` to take `num_workers` as an optional argument (with a default value of 2) and overriding that default in a new test case `test_distributed`

Reply:
The unit test function was added